cameronRomo/whats-for-dinner
# Feature/build out comp

What’s this PR do?
* Nearly completes the functionality to IT 1
  - Need to fix overlapping DOM elements

Where should the reviewer start?
* displayRandom()

How should this be manually tested?
* This could be tested by deploying the site and clicking either side or main meal
  - To see bug click side, then lets cook, then meal without clearing.

Any background context you want to provide?
* DOM manipulation

What are the relevant tickets?
* N/A

Screenshots (if appropriate)
